### PR TITLE
fix update with id=0

### PR DESCRIPTION
**Pull request by valebar**
This should fix the issue when you edit a record with id = 0

### CODE REVIEW

File: src/app/Library/CrudPanel/Traits/Read.php [hunk 1]
```diff
@@ -40,8 +40,10 @@ public function getCurrentEntry()
     {
         $id = $this->getCurrentEntryId();
 
-        if (! $id) {
-            return false;
+        if (is_bool($id)) {
+            if (! $id) {
+                return false;
+            }
         }
```
Comment (tabacitu) [inline src/app/Library/CrudPanel/Traits/Read.php:47]:
```suggestion
        if ($id === false) {
                return false;
        }
```

I'm wondering - would this do the same thing @valebar ? It looks like it to me, and it'd be a little cleaner right?

**Review by tabacitu — CHANGES_REQUESTED** [review]

File: src/app/Library/CrudPanel/Traits/Read.php (hunk 1 above)
Comment (valebar) [reply src/app/Library/CrudPanel/Traits/Read.php]: sorry... I accidently pushed the code you suggested :) mea culpa

Comment (tabacitu) [reply src/app/Library/CrudPanel/Traits/Read.php]: No problem at all - so did you get to test it, works well in your situation?

Comment (valebar) [reply src/app/Library/CrudPanel/Traits/Read.php]: it works as expected.

**Comment by tabacitu** [comment]
Great, thanks for the PR and the confirmation @valebar ! Merging now, it'll be available with a `composer update` on Monday.

Cheers!

**Pull request merged**